pkgs/applications/editors/eclipse/plugins.nix

```diff
@@ -639,4 +639,92 @@ rec {
     };
   };
 
+  ivyde = buildEclipsePlugin rec {
+    name = "ivyde-${version}";
+    version = "2.2.0.final-201311091524-RELEASE";
+
+    srcFeature = fetchurl {
+      url = "https://downloads.apache.org/ant/ivyde/updatesite/ivyde-${version}/features/org.apache.ivyde.feature_${version}.jar";
+      sha1 = "c8fb6c4aab32db13db0bd81c1a148032667fff31";
+    };
+
+    srcPlugin = fetchurl {
+      url = "https://downloads.apache.org/ant/ivyde/updatesite/ivyde-${version}/plugins/org.apache.ivyde.eclipse_${version}.jar";
+      sha1 = "0c80c2e228a07f18efab1c56ea026448eda70c06";
+    };
+
+    meta = with lib; {
+      homepage = "https://ant.apache.org/ivy/ivyde/index.html";
+      description = "A plugin which integrates Apache Ivy's dependency management";
+      license = licenses.asl20;
+      platforms = platforms.all;
```

Comment:
meta should always contain a maintainer.

Reply:
As mentioned before, it was my first pull. So I didn't yet feel suitable as maintainer. But now I think I can add myself.
Btw. many other plugins also haven't a maintainer and some of them didn't work anymore. Should they be removed?

Reply:
> Btw. many other plugins also haven't a maintainer and some of them didn't work anymore. Should they be removed?

If there is an update that makes them work again they should be updated but if upstream no longer develops them and there are no fixes we can drop them.